src/legacy/core_plugins/kibana/common/field_formats/types/url.ts

```diff
@@ -0,0 +1,191 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { escape, memoize } from 'lodash';
+import {
+  getHighlightHtml,
+  FieldFormat,
+  TEXT_CONTEXT_TYPE,
+  HTML_CONTEXT_TYPE,
+} from '../../../../../../plugins/data/common/';
+
+const templateMatchRE = /{{([\s\S]+?)}}/g;
+const whitelistUrlSchemes = ['http://', 'https://'];
+
+const URL_TYPES = [
+  { kind: 'a', text: 'Link' },
+  { kind: 'img', text: 'Image' },
+  { kind: 'audio', text: 'Audio' },
+];
+const DEFAULT_URL_TYPE = 'a';
+
+export function createUrlFormat() {
+  class UrlFormat extends FieldFormat {
+    static id = 'url';
+    static title = 'Url';
+    static fieldType = [
+      'number',
```

Comment:
It's a KBN_FIELD_TYPES. Please replace it

Reply:
Replaced. Makes sense. Thanks